src/bridge/setup.rs

```diff
@@ -146,7 +146,6 @@ pub async fn setup_neovide_specific_state(
         .ok();
 }
 
-#[cfg(windows)]
```

Comment:
Was this change intentional? It yields an unused warning on Linux.

Reply:
Oh actually, yes and no. Earlier I did that because I was creating a new `:NeovideMinimize` command however that idea was scratched off and I reverted those changes. However it seems I forgot to revert this specific change.

It is not required and the attribute should be added back.

Reply:
I'll add it back then!